src/Resources/views/Core/tab_menu_template.html.twig

```diff
@@ -120,7 +120,8 @@
 
 {% block label %}
 {{-
-    item.label|trans(
+    # We use method accessor instead of ".label" since `item` implements `ArrayAccess` and could have a property called "label".
```

Comment:
```suggestion
    {# We use method accessor instead of ".label" since `item` implements `ArrayAccess` and could have a property called "label". #}
```